src/frontend/src/pages/FinancePage/ReimbursementRequestsSection.tsx

```diff
@@ -1,23 +1,31 @@
 import { Box, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, useTheme } from '@mui/material';
 import { Link as RouterLink } from 'react-router-dom';
 import { useState } from 'react';
-import { ReimbursementRequest, isAdmin } from 'shared';
+import { ReimbursementRequest, isAdmin, Vendor, ReimbursementStatusType } from 'shared';
 import { useCurrentUser } from '../../hooks/users.hooks';
 import { centsToDollar, datePipe, dateUndefinedPipe, fullNamePipe, undefinedPipe } from '../../utils/pipes';
 import ColumnHeader from './FinanceComponents/ColumnHeader';
 import FinanceTabs from './FinanceComponents/FinanceTabs';
 import { routes } from '../../utils/routes';
 import { cleanReimbursementRequestStatus, createReimbursementRequestRowData } from '../../utils/reimbursement-request.utils';
+import TableSortLabel from '@mui/material/TableSortLabel';
 
 interface ReimbursementRequestTableProps {
   userReimbursementRequests: ReimbursementRequest[];
   allReimbursementRequests?: ReimbursementRequest[];
 }
 
+type ColumnNames = 'id' | 'saboId' | 'amount' | 'dateSubmitted' | 'status' | 'dateSubmittedToSabo' | 'submitter' | 'vendor';
```

Comment:
I think we can make this type definition separately in `FinishLine\src\shared\src\types\reimbursement-requests-types.ts`
as an `enum` type. We can then import it in this file to use it.